### PR TITLE
A : [NSFW] videosdesuavizinha.com

### PR DESCRIPTION
**Pull request by felippesantos46**
Blocking and Hiding Filters for [videosdesuavizinha](http://videosdesuavizinha.com/) [NSFW]

The filters were requested at the issue #https://github.com/easylist/easylistportuguese/issues/73 as i do not have permissions to close i would like to ask for the evaluation of the following filters:

Blocking : `||cameracaseira.com^$script,domain=videosdesuavizinha.com`
Hiding : `videosdesuavizinha.com##a[title^="Acompanhantes"]`

PS: The screenshot is inside the issue mentioned above.

### CODE REVIEW

File: easylistportuguese/easylistportuguese_specific_block.txt [hunk 1]
```diff
@@ -97,4 +97,5 @@
 ||ecoregional.com.br/wp-content/uploads/*/jardim-pq-*.jpg
 ||ecoregional.com.br/wp-content/plugins/icegram/
 ||notebookcheck.info/fileadmin/Sonstiges/ama_abn2_$xmlhttprequest
-||noticiasdecoimbra.pt/*/modal/*/modal.min*^$stylesheet,script
\ No newline at end of file
+||noticiasdecoimbra.pt/*/modal/*/modal.min*^$stylesheet,script
+||i1.wp.com/www.cenariomt.com.br/wp-content/uploads/*/provincia-at.png
```
Comment (jabonsolo) [inline easylistportuguese/easylistportuguese_specific_block.txt:101]: @felippesantos46  could you remove this filter from this PR, it belongs to a different one.

https://github.com/easylist/easylistportuguese/pull/76

File: easylistportuguese/easylistportuguese_specific_hide.txt [hunk 2]
```diff
@@ -256,4 +256,9 @@ tecnoblog.net##div[id^="fullbanner-container"]
...
 tecnoblog.net##div[id^="company-header-ad"]
 tecnoblog.net##section[id^="ad-superbanner-"]
-br.ccm.net,saude.ccm.net##div[class^="ads--insertor-"]
\ No newline at end of file
+br.ccm.net,saude.ccm.net##div[class^="ads--insertor-"]
+cenariomt.com.br##a[href^="https://www.inexa.com.br"]
+cenariomt.com.br##a[href^="http://www.agrobaggio.com.br"]
+cenariomt.com.br##a[data-img-url^="https://www.cenariomt.com.br/wp-content/uploads/2021/07/amazoniajuho2021.gif"]
+cenariomt.com.br##a[href^="http://encurtador.com.br"]
+cenariomt.com.br##a[href^="https://www.modelocontabilidade.com"]
```
Comment (jabonsolo) [inline easylistportuguese/easylistportuguese_specific_hide.txt:264]: These filters also belong to a different PR
https://github.com/easylist/easylistportuguese/pull/76

**Comment by jabonsolo** [comment]
Apart of the comments I added, @felippesantos46  could you also add this filter
`||parlorscenes.com^$script`

This blocks  for me popups when clicking on any part of screen.

**Comment by felippesantos46** [comment]
My bad, i got confused ... I believe now we are good to go !
If something, just let me know.
